be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -7,8 +7,8 @@
 namespace starrocks::pipeline {
 
 Status AggregateBlockingSinkOperator::prepare(RuntimeState* state) {
-    RETURN_IF_ERROR(Operator::prepare(state));
```

Comment:
I think you needn't change the `RETURN_IF_ERROR` method name. you could make your change in `RETURN_IF_ERROR ` method internally.    https://github.com/StarRocks/starrocks/pull/5434/files why not use libfiu?

Reply:
> 1 I think you needn't change the `RETURN_IF_ERROR` method name. you could make your change in `RETURN_IF_ERROR ` method internally. 2 https://github.com/StarRocks/starrocks/pull/5434/files why not use libfiu?

We can't replace every use of RETURN_IF_ERROR with RETURN_IF_WITH_INJECT due to factors:
- starrocks_main.cpp: replace this range will cause failed to start.
- others(linclude storage directory, features like compactions/Import...): this range is not our tearget range.